javascript/javascript2/week3/functionAsObjectMember.js

```diff
@@ -0,0 +1,7 @@
+const functionInObject = {
```

Comment:
Probably it's better to have a more meaningful class like `Student` and have a function that returns the full name